app/controllers/admin/api_docs/services_controller.rb

```diff
@@ -1,10 +1,9 @@
 class Admin::ApiDocs::ServicesController < FrontendController
-  before_action :find_service, :only => [:destroy, :edit, :update, :show, :preview, :toggle_visible]
+  before_action :find_api_docs, only: %i[destroy edit update show preview toggle_visible]
+  before_action :set_service_navigation_if_service, only: %i[edit preview]
```

Comment:
If the service has many ActiveDocs, then we not showing the index scoped?

Reply:
@guicassolato This controller is not per service, it is per account, we have here the index of all api_docs_services of an account, and there is another controller for the index of all api_docs_services of a service. Because we still want to see both views 😄

Reply:
Probably too much work and not worth it, but it would be nice we could use 
https://github.com/3scale/porta/blob/6f74cf39b5bfef598ef24388b17fb98ad3618484/lib/menu_system.rb#L54

```ruby
activate_menu service_navigation: true
```

Reply:
I don't think it is worth it for now that we only want it for 1 controller, but I will definitely keep it in mind for the future if we need it somewhere else 😄